crates/floresta-electrum/src/electrum_protocol.rs

```diff
@@ -400,28 +406,32 @@ impl<Blockchain: BlockchainInterface> ElectrumServer<Blockchain> {
 
         self.wallet_notify(&transactions).await;
     }
+
+    /// Handles each kind of Message
     async fn handle_message(&mut self, message: Message) -> Result<(), crate::error::Error> {
         match message {
-            Message::NewPeer((id, stream)) => {
-                self.peers.insert(id, stream);
+            Message::NewClient((id, client)) => {
+                self.clients.insert(id, client);
             }
-            Message::Message((peer, msg)) => {
+
+            Message::Message((client, msg)) => {
                 trace!("Message: {msg}");
                 if let Ok(req) = serde_json::from_str::<Request>(msg.as_str()) {
-                    let peer = self.peers.get(&peer);
-                    if peer.is_none() {
+                    let client = self.clients.get(&client);
+                    if client.is_none() {
                         log!(
```

Comment:
Since you're touching it already, it might make sense to remove these explicit invocations to `log` and use `error!()` and `info!()`. This codebase annoyingly uses both.

Reply:
done